src/all/exhentai/src/eu/kanade/tachiyomi/extension/all/exhentai/ExGalleryMetadata.kt

```diff
@@ -0,0 +1,53 @@
+package eu.kanade.tachiyomi.extension.all.exhentai
+
+import android.net.Uri
+
+/**
+ * Gallery metadata storage model
+ */
+
+class ExGalleryMetadata {
+    var url: String? = null
+
+    var thumbnailUrl: String? = null
+
+    var title: String? = null
+    var altTitle: String? = null
+
+    var genre: String? = null
+
+    var datePosted: Long? = null
+    var parent: String? = null
+    var visible: String? = null // Not a boolean
```

Comment:
Comment seems useless, with the type annotation?

Reply:
I would consider this: // Deliberately not boolean

Not seen them using code, but with nullable, you have three states on a boolean.